spark8t/domain.py

```diff
@@ -53,6 +53,9 @@ def _read_property_file_unsafe(cls, name: str) -> Dict:
         defaults = dict()
         with open(name) as f:
             for line in f:
+                # skip empty line
+                if len(line.strip()) == 0:
```

Comment:
**suggestion** Maybe this can be factored out in a function, such that extra logic (like "commented line filtering) can be easily extended. Right now we would need to change that in three other places